ingestion/src/metadata/ingestion/source/database/bigquery/metadata.py

```diff
@@ -615,26 +615,27 @@ def get_schema_definition(
         Get the DDL statement or View Definition for a table
         """
         try:
-            if table_type == TableType.View:
-                view_definition = inspector.get_view_definition(
-                    fqn._build(self.context.get().database, schema_name, table_name)
+            if self.source_config.includeDDL:
```

Comment:
We should do that check where we call `get_schema_definition` so `get_schema_definition` actually get only the schema definition